src/index.ts

```diff
@@ -89,12 +90,13 @@ EventEmitter.captureRejections = true;
         })
       );
 
-      const slashCommandUploads = Array.from(slashCommands.entries()).map(([key, command]) => {
+      const slashCommandUploads = Array.from(commands.entries()).map(([key, command]) => {
         return {
+          type: command.type,
           name: key,
-          description: command.description.substring(0, 99),
-          options: command.options,
-        };
+          description: 'description' in command ? command.description.substring(0, 99) : undefined,
```

Comment:
Does this not work with just 
```
command.description?.substring(0, 99)
```
does that not do the same, but return undefined if its not specified?

Reply:
These are mainly here to make the ts compiler happy, doing `command.description` reports an error because a context menu has no description.

Reply:
Actually I found a way around this.